doc/en/components/charts/features/chart-highlight-filter.md

```diff
@@ -0,0 +1,63 @@
+
+By default, the data source used for the `HighlightedValueMemberPath` will be what you have set the `ItemsSource` of the series or chart to, but you can also provide your own by using the `HighlightedItemsSource` property of the series. Note, that if you are using a category series, the schema of the `HighlightedItemsSource` will need to match the schema of the `ItemsSource` property of that series.
+
+The part of the series shown by the highlight filter will be represented in the legend and tooltip layers of the chart separately. You can configure the title that this is given in the tooltip and legend by setting the `HighlightedTitleSuffix`. This will append the value that you provide to the end of the `Title` of the series.
+
+The following example demonstrates the usage of this feature within the `XamDataChart` control:
+
+`sample="/charts/data-chart/chart-highlight-filter", height="500", alt="{Platform} Highlight Filter Example"`
+
+## Differences in CategoryChart
+
+The `CategoryChart` highlight filter happens on the chart by setting the `InitialHighlightFilter` property. Since the `CategoryChart` takes all of the properties on your underlying data item into account by default, you will need to define the `InitialGroups` on the chart as well so that the data can be grouped and aggregated in a way that you can have a subset of the data to filter on. You can set the `InitialGroups` to a value path in your underlying data item to group by a path that has duplicate values. 
+
+<!-- Unsure of this part. Need to review -->
+<!-- ????? The `InitialHighlightFilter` is done using OData filter query syntax. The syntax for this is an abbreviation of the filter operator. For example, if you wanted to have an InitialHighlightFilter of "Month not equals January" it would be represented as "Month ne 'January'"-->
+
+Like the `XamDataChart`, the `HighlightedValuesDisplayMode` property is also exposed on the `CategoryChart` in the case that you do not want to see the reduced opacity part of the highlight filter layer.
+
+The following example demonstrates the usage of this feature within the `CategoryChart` control:
+
+`sample="/charts/category-chart/chart-highlight-filter", height="500", alt="{Platform} Highlight Filter Example"`
+
+## Additional Resources
+
+You can find more information about related chart features in these topics:
+
+- [Chart Highlighting](chart-highlighting.md)
+- [Chart Data Tooltip](chart-data-tooltip.md)
+- [Chart Data Aggregations](chart-data-aggregations.md)
+
+## API References
+
+The following is a list of API members mentioned in the above sections:
+
+- `HighlightedItemsSource`
+- `HighlightedTitleSuffix`
+- `HighlightedValueMemberPath`
+- `HighlightedValuesDisplayMode`
+- `HighlightedValuesFadeOpacity`
+- `HighlightedValuesDisplayMode`
+- `InitialHighlightFilter`
+- `InitialGroups`
+- `CategoryChart`
+- `XamDataChart`
```

Comment:
can you change this list to a table with 2 columns for category chart and data chart?
also, try using class prefixes to resolve ambiguity between Series properties and Category Chart properties:
`Series.HighlightedItemsSource` and 

`DomainChart.HighlightedItemsSource` or
`CategoryChart.HighlightedItemsSource`